[2025-01-10T15:03:29.685+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2025-01-10T15:03:23+00:00 [queued]>
[2025-01-10T15:03:29.703+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2025-01-10T15:03:23+00:00 [queued]>
[2025-01-10T15:03:29.704+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-01-10T15:03:29.752+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): runme_2> on 2025-01-10 15:03:23+00:00
[2025-01-10T15:03:29.760+0000] {standard_task_runner.py:57} INFO - Started process 240 to run task
[2025-01-10T15:03:29.766+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2025-01-10T15:03:23+00:00', '--job-id', '203', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpqqi7da_f']
[2025-01-10T15:03:29.779+0000] {standard_task_runner.py:85} INFO - Job 203: Subtask runme_2
[2025-01-10T15:03:29.873+0000] {task_command.py:415} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2025-01-10T15:03:23+00:00 [running]> on host f9fbff8a64e9
[2025-01-10T15:03:30.737+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T15:03:23+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-10T15:03:23+00:00'
[2025-01-10T15:03:30.741+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-10T15:03:30.742+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20250110" && sleep 1']
[2025-01-10T15:03:30.765+0000] {subprocess.py:86} INFO - Output:
[2025-01-10T15:03:30.784+0000] {subprocess.py:93} INFO - example_bash_operator__runme_2__20250110
[2025-01-10T15:03:31.795+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-10T15:03:31.826+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20250110T150323, start_date=20250110T150329, end_date=20250110T150331
[2025-01-10T15:03:31.872+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-10T15:03:31.914+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
